eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/RatifySpec.hs

```diff
@@ -1069,6 +1069,18 @@ votingSpec =
             passNEpochs 2
             -- The same vote should now successfully ratify the proposal
             getLastEnactedCommittee `shouldReturn` SJust (GovPurposeId addCCGaid)
+        it "DRepAlwaysNoConfidence is sufficient to pass NoConfidence" $ do
```

Comment:
I know this test with all the nested levels is already kinda unwieldy, but I wonder if moving this test under  `Predefined DReps` tests above would make more sense than having it under `StakePool` ?

Reply:
Yeah, agreed